src/Core/Services/Implementations/SsoConfigService.cs

```diff
@@ -23,6 +34,35 @@ public async Task SaveAsync(SsoConfig config)
             {
                 config.CreationDate = now;
             }
+
+            if (config.GetData().UseKeyConnector)
```

Comment:
`GetData()` deserializes, should we assign these calls to local variables to avoid repeatedly deserializing the same data, or is C# smart enough to cache the results?